### PR TITLE
outpost tweaks & fixes: Vengeance 

### PR DESCRIPTION
**Pull request by goober3**
replica of #2230
## About The Pull Request
both outposts have been passed for mistakes, such as wiring & pipe errors.

- the indie outpost no longer runs out of power immediately (the smes didn't have a terminal).
- a one-way maintenance door has been added to the indie outpost disposal room, so players aren't sealed forever.
- doors to inaccessible areas in the indie outpost are now sealed with doors + walls, rather than re-iconned walls.
___
- the ice asteroid outpost's map has been expanded, to avoid windows to void.
- some doors have been replaced, to better fit anywalls.
- freezer room floor tiles have been replaced with showroom floors.

outpost & hangar files now have unique names + their faction.
all wallmounts + doors are now fixed & use proper dirs.
the nt asteroid outpost now has a 56x40 hangar.
hangar tiles now use planetary atmos, to prevent depressurization via ship.
the floor under elevator doors on all hangars is now catwalk. 

## Why It's Good For The Game

bugfixes are good. no more floating extinguishers & unbreathable elevators.

## Changelog

:cl:
add: the nt asteroid outpost now has a 56x40 hangar.
fix: outpost hangars should no longer depressurize upon ship docking.
fix: outpost wallmounts and doors have been fixed. please report any floating extinguishers & hovering intercoms.
fix: the independent outpost should stay powered forever.
/:cl:

### CODE REVIEW

File: _maps/outpost/hangar/indie_space_20x20.dmm [hunk 1]
```diff
@@ -1,79 +1,89 @@
...
 /obj/machinery/light/floor/hangar,
-/turf/open/floor/plasteel,
+/turf/open/floor/plasteel{
+	planetary_atmos = 1
+	},
 /area/hangar)
 "e" = (
 /obj/effect/landmark/outpost/hangar_numbers,
-/turf/open/floor/plasteel,
+/turf/open/floor/plasteel{
```
Comment (tmtmtl30) [inline _maps/outpost/hangar/indie_space_20x20.dmm:19]: Not a big fan of these changes without altering the hangar baseturfs as well. I'm pretty sure that, for instance, once this floor is removed, the plating underneath won't have planetary atmos. More critically, if that *plating* is removed (such as by explosion), the space underneath will suck up all the air created by the other planetary turfs in the hangar, creating a perpetual pressure difference. That seems quite bad.

**Review by tmtmtl30 — DISMISSED** [review]

File: _maps/outpost/hangar/indie_space_20x20.dmm (hunk 1 above)
Comment (goober3) [reply _maps/outpost/hangar/indie_space_20x20.dmm:19]: ruh roh

Comment (goober3) [reply _maps/outpost/hangar/indie_space_20x20.dmm:19]: i'm not sure I could fix this without making unique subtypes for every type of floor used. shame!!

Comment (Erikafox) [reply _maps/outpost/hangar/indie_space_20x20.dmm:19]: better sooner than later

File: _maps/outpost/hangar/indie_space_20x20.dmm [hunk 2]
```diff
@@ -103,47 +119,64 @@
...
+	planetary_atmos = 1
+	},
 /area/hangar)
 "K" = (
 /obj/effect/turf_decal/arrows{
 	dir = 1
 	},
-/turf/open/floor/plasteel,
+/turf/open/floor/plasteel{
+	planetary_atmos = 1
```
Comment (Erikafox) [inline _maps/outpost/hangar/indie_space_20x20.dmm:140]: still gotta make these a subtype

**Review by spockye — APPROVED** [review]

File: _maps/outpost/indie_space.dmm
Comment (GenericDM) [inline _maps/outpost/indie_space.dmm:1]: image
Floating newscaster in cargo.

**Comment by GenericDM** [comment]
Indestructible reinforced walls which are used in the outpost do not smooth properly with doors and some windows. This can be fixed by changing lines 85 and 86 in code\game\turfs\closed\_closed.dm to include the correct smoothing flags, like so:
`smoothing_groups = list(SMOOTH_GROUP_CLOSED_TURFS, SMOOTH_GROUP_WALLS, SMOOTH_GROUP_AIRLOCK)`
`canSmoothWith = list(SMOOTH_GROUP_WALLS, SMOOTH_GROUP_WINDOW_FULLTILE, SMOOTH_GROUP_AIRLOCK)`

**Comment by meemofcourse** [comment]
did you know? you can leave comments on files to suggest these changes

**Comment by thgvr** [comment]
we need to merge this god

**Comment by MarkSuckerberg** [comment]
I've been waiting for them to be made a subtype but I guess it's fine as is :shrivel:

**Pull request merged**